tests/vmi_configuration_test.go

```diff
@@ -505,6 +505,20 @@ var _ = Describe("[sig-compute]Configurations", func() {
 		})
 
 		Context("with BIOS bootloader method and no disk", func() {
+			withSerialBIOS := func() libvmi.Option {
```

Comment:
when possible, please use static `func` definitions. They reduce mental burden. If they are well-named and placed at the bottom of the file, I do not need to read their content at all to understand the test.

Reply:
Done.